src/extensionApi.ts

```diff
@@ -92,16 +93,25 @@ export class CommandHandler {
             selectedServerType = context.server.type;
         }
 
+        const remove = await vscode.window.showWarningMessage(
+            `You are about to remove server ${serverId}.\nAre you sure?`, { modal: true }, 'Yes, remove');
```

Comment:
The same here. There is no need to ask 'Are you sure?` IMO. Simply ask: "Do you want to remove server 'serverName'?" and give two options 'Yes', 'Cancel'

Reply:
even shorter: ```Remove server <servername>?"